docs/users-guide/building.rst

```diff
@@ -84,7 +84,7 @@ The following example shows the steps for the case where the |Viskores| source i
    cmake-gui ../viskores-v2.1.0
 
 .. _fig:CMakeGUI:
-.. figure:: images/CMakeGUIBoth.png
+.. figure::  ../../data/users-guide/images/CMakeGUIBoth.png
```

Comment:
Hmm. It's kind of awkward to reference images here. Maybe eventually we will make a second submodule for these images. But let's think about that and worry about it later.

Reply:
@kmorel thanks for the prompt review, I think that this is ready to go. I will merge it.